Main/Model/Ghost.py

```diff
@@ -2,14 +2,30 @@
 
 
 class Ghost(DynamicEntity):
-    def __init__(self, color):
+    def __init__(self,movement_strategy,color):
+        DynamicEntity.__init__(self,movement_strategy)
         self.color = color
-        
+
+    def eat(self,entity):
+        entity.kill()      
+
+    def get_direction(self):
+        return self.get_strategy()
+
+    def get_strategy(self):
```

Comment:
```suggestion
    def get_movement_strategy(self):
```

Igual estos 2 metodos no deberían estár en Ghost... son de DynamicEntity

Reply:
claro, el get_strategy no deberías usarlo nunca, lo podes eliminar.
y el get_direction ya lo tenes en el padre, así que lo podes eliminar tb